sdk/network/azure-mgmt-network/HISTORY.rst

```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+6.0.0 (2019-10-09)
+++++++++++++++++++
+
```

Comment:
just for consistency, there should be empty lines before and after **Features** and **Breaking Changes**

Reply:
in my local vi, when line number is off, the lines seem to be correct. but if i do `:set number`, the format is a mess, the same as here.